bitnami/contour/ci/ct-values-deployment.yaml

```diff
@@ -0,0 +1 @@
+kind: Deployment
```

Comment:
Please also add block below since the cluster used in the GitHUB CI/CD has no support for LoadBalancer services:
```yaml
envoy:
  service:
    type: ClusterIP
    externalTrafficPolicy: null
```